CHANGELOG.md

```diff
@@ -1,5 +1,15 @@
 # Change Log
 
+## 1.7.0 - 2022-05-24
+
+<!-- insert link to app centric changelog -->
```

Comment:
Guess there's not much point in me approving until this link is updated.

Reply:
I created an aka.ms link that points to the RGs changelog for now. We'll update it when we have something real to point it at.